script/Arcueid/Day 7/07_00_ARC07_1_2.txt

```diff
@@ -91,32 +102,37 @@ There wasn't anything remotely close to an 'animalistic instinct'.
 [sha:7455cf5191df92c10e539ff68ab8f521b81c990d]{
 -- Page 140, Offset 16923.
 -- もうまるっきりしなかった。
-Such premonitions and hopes never came to light.
+All my hunches and hopes inevitably ended in disappointment.
 }
 [sha:0d7f360a959b96fda5ee838dfa5c78ccf723b383]{
 -- Page 141, Offset 16924.
 -- ここまで清々しい無駄骨は久しぶりだ。言い過ぎかもしれないが初めてですらある。
-It's been a while since I wasted my time so magnificently like this. I might even go as far as to say that this is the first.
+I can't remember the last time I went on such a refreshing fool's errand.
+ I might even go so far as to say this is a first.
 }
 [sha:5cb8447561154ca98311d64abd49634f9098e4e7]{
 -- Page 141, Offset 16925.
 -- いつ貧血が起きるか分からないため、遠野志貴は『無駄な行動』をしないよう努めていたからだ。
-That's because I've tried hard not to overexert myself out of concern that my anemia could well up at any moment.
+I suppose the reason for that is because I never knew when my anemia might flare up,
+ and was always careful to avoid such meaningless exertion as best as I was able.
```

Comment:
"when my anemia would flare up; I've always been careful to avoid meaningless exertion as best I could"